src/stratigraphy/util/dataclasses.py

```diff
@@ -16,6 +18,9 @@ class Point:
     def tuple(self) -> (float, float):
         return self.x, self.y
 
+    def distance_to(self, point: Point) -> float:
```

Comment:
Where not using this at all, are we? Happy to keep it anyways as it's a logic extension of this class.